frontend/pages/admin/AdminWrapper.tests.tsx

```diff
@@ -0,0 +1,183 @@
+import React from "react";
+import { screen } from "@testing-library/react";
+import { createCustomRenderer } from "test/test-utils";
+
+import { noop } from "lodash";
+
+import createMockUser from "__mocks__/userMock";
+import createMockConfig from "__mocks__/configMock";
+import createMockTeam from "__mocks__/teamMock";
+
+import AdminWrapper from "./AdminWrapper";
+
+const urlLocation = {
+  pathname: "settings/organization/info",
+};
+
+// TODO: figure out how to mock the router properly.
+const mockRouter = {
+  push: jest.fn(),
+  replace: jest.fn(),
+  goBack: jest.fn(),
+  goForward: jest.fn(),
+  go: jest.fn(),
+  setRouteLeaveHook: jest.fn(),
+  isActive: jest.fn(),
+  createHref: jest.fn(),
+  createPath: jest.fn(),
+};
+
+describe("AdminWrapper - component", () => {
```

Comment:
Nice. Do these replace any QA wolf tests, or just good additions? The screenshot only lists topnav-related tests.